### PR TITLE
Simple Mob Blood Revive Fix

### PR DESCRIPTION
**Pull request by Geevies**
* Reviving simple mobs will now restore their blood.

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 1]
```diff
@@ -171,6 +171,11 @@
 	turns_since_move = turns_per_move
 	..()
 
+/mob/living/simple_animal/revive(reset_to_roundstart)
+	. = ..()
+	blood_amount = initial(blood_amount)
+	bleeding = FALSE
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/simple_animal/simple_animal.dm:177]: Is this properly called when rejuvenate is called on a living mob, and does this clear the blood overlays?

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/simple_animal.dm (hunk 1 above)
Comment (Geevies) [reply code/modules/mob/living/simple_animal/simple_animal.dm:177]: yes

**Review by JohnWildkins — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**